src/cmdlib.sh

```diff
@@ -279,7 +279,7 @@ EOF
     rootfs_overlay="${overridesdir}/rootfs"
     if [ -d "${rootfs_overlay}" ]; then
         echo "Committing ${rootfs_overlay}"
-        sudo ostree commit --repo="${workdir}/tmp/repo" --tree=dir="${rootfs_overlay}" -b cosa-bin-overlay \
+        ostree commit --repo="${tmprepo}" --tree=dir="${rootfs_overlay}" -b cosa-bin-overlay \
```

Comment:
Hmm, I started out without the `sudo` and hit an issue, but I forget what it was.  If you tested this I'm OK with it.

Reply:
Yeah, working fine here at least.